### DISCUSSION TITLE
Archetype Directory Bundle

### DISCUSSION THREAD

**Discussion by technosprout** [thuliteio/doks#1059] [General]
Looking for a solution to create the following directory structure (directory-bundle)
archetype > articles 
                     _index.md
                      /media
                              /img.jpg

The idea is to create a local media folder so all images with that article are stored in a subfolder
The media folder contents should not show on the final content page
but the article.md should be able to reference the folder as media/img.jpg

content > articles >new-article
                                new-article.md
                                /media
                                           /img.jpg

**Comment by h-enk** [comment]
Unfortunately, does not work with Doks v0.5.

However, it will work with Doks 1.0 (out of the box). Tested and confirmed with Doks [v1.0.0-alpha.1](https://github.com/h-enk/doks/releases/tag/v1.0.0-alpha.1).

Know that I'm pushing on releasing Doks 1.0 ASAP 💥

**Comment by ueda19850603** [comment]
This article was helpful
https://discourse.gohugo.io/t/images-are-not-loading-in-doks-theme/44257/4
